pkg/sql/conn_executor.go

```diff
@@ -4005,7 +4006,9 @@ func (ex *connExecutor) handleWaitingForConcurrentSchemaChanges(
 ) error {
 	if err := ex.planner.waitForDescriptorSchemaChanges(
 		ctx, descID, *ex.extraTxnState.schemaChangerState,
-	); err != nil {
+	); err != nil &&
```

Comment:
the original PR had a helpful comment here. i think we should keep it
```
	// If we encountered a missing or dropped / offline descriptor waiting for the schema
	// change then lets ignore the error, and let the FSM retry, since concurrentSchemaChangeError
	// errors are retryable. Otherwise, allow the error to bubble back up and kill
	// the connection.
```

Reply:
I forgot to copy the comment over, let me do that